### PR TITLE
Bump common def, common utils and server in client

### PR DESCRIPTION
**Pull request by anthony-murphy**
needed for #5887

### CODE REVIEW

File: examples/apps/likes-and-comments/package.json [hunk 1]
```diff
@@ -33,12 +33,12 @@
...
     "@fluid-experimental/get-container": "^0.39.0",
     "@fluidframework/aqueduct": "^0.39.0",
-    "@fluidframework/common-definitions": "^0.19.1",
+    "@fluidframework/common-definitions": "^0.20.0-0",
     "@fluidframework/counter": "^0.39.0",
     "@fluidframework/datastore-definitions": "^0.39.0",
     "@fluidframework/map": "^0.39.0",
     "@fluidframework/merge-tree": "^0.39.0",
-    "@fluidframework/protocol-definitions": "^0.1022.0",
+    "@fluidframework/protocol-definitions": "^0.1024.0-0",
```
Comment (heliocliu) [inline examples/apps/likes-and-comments/package.json:41]: @tanviraumi are there any server changes we should be actively aware of when taking this newer version?  It seems a bit off that we managed to jump over a version here

**Review by heliocliu — APPROVED** [review]
signing off assuming no issues around the server version jump and fixing the compile issues

File: examples/apps/likes-and-comments/package.json (hunk 1 above)
Comment (tanviraumi) [reply examples/apps/likes-and-comments/package.json:41]: I don't recall any major changes in protocol. So it should be fine. We have also started to maintain a BREAKING.md for server (https://github.com/microsoft/FluidFramework/blob/main/server/BREAKING.md).

**Review by tanviraumi — APPROVED** [review]

**Comment by anthony-murphy** [comment]
currently blocked on https://github.com/microsoft/FluidFramework/pull/5908

**Pull request merged**